torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -93,15 +93,12 @@ bool bothMinMaxNone(
   const auto& match_vmap = match.values_map;
   // If value map has dummy_min_max in it, then it means
   // the pattern to be replaced was the one containing hardtanh.
-  if (vmap.find("dummy_min_max") != vmap.end()) {
-    auto dummy_min_max =
-      graph_rewrite_helper::getIValue("dummy_min_max", match_vmap, vmap);
-    return (!dummy_min_max || dummy_min_max.value().isNone());
-  }
-  // Otherwise the pattern is relu.
-  auto output_min_max =
-    graph_rewrite_helper::getIValue("output_min_max", match_vmap, vmap);
-  return (!output_min_max || output_min_max.value().isNone());
+  TORCH_CHECK(
+      vmap.find("dummy_min_max") != vmap.end(),
+      "Expected to find dummy_min_max Value in the subgraph to be replaced.");
+  auto dummy_min_max =
+    graph_rewrite_helper::getIValue("dummy_min_max", match_vmap, vmap);
+  return (!dummy_min_max || dummy_min_max.value().isNone());
```

Comment:
Is `!dummy_min_max` still valid?

Reply:
Just guarding against the possibility that instead of None Value, getIValue returns c10::nullopt.